content/docs/SUSHI/configuration/exhaustive-config.yaml

```diff
@@ -124,6 +136,10 @@ groups:
 # file name extension.  Any subproperties that are valid filenames
 # with supported extensions (e.g., .md/.xml) will be treated as
 # sub-pages.
+# For R5 ImplementationGuides, the IG.definition.page element now
```

Comment:
I think a blank line would be nice here as well.  But with a `#` since it's still in context of the previous comments.
```suggestion
#
# For R5 ImplementationGuides, the IG.definition.page element now
```